webpack.config.js

```diff
@@ -13,16 +14,17 @@ module.exports = [
             libraryTarget: 'umd',
             path         : path.resolve(__dirname, 'dist'),
         },
-        module: {
-            rules : [
+        externals: [nodeExternals()], // fix https://github.com/websockets/ws/issues/1126
```

Comment:
Is this done to ignore `node_modules` when bundling?.
We bundle it for FE too, we should be including node_modules libraries.

Reply:
I made a mistake, will try to remove it again.